lib/cocoapods/sandbox/podspec_finder.rb

```diff
@@ -12,13 +12,9 @@ def podspecs
         @specs_by_name = {}
         spec_files = Pathname.glob(root + '{,*}.podspec{,.json}')
         spec_files.sort_by { |p| -p.to_path.split(File::SEPARATOR).size }.each do |file|
-          begin
-            spec = Specification.from_file(file)
-            spec.validate_cocoapods_version
-            @specs_by_name[spec.name] = spec
-          rescue => e
-            UI.warn "Unable to load a podspec from `#{file.basename}`, skipping:\n\n#{e}"
```

Comment:
Well the more I think about this maybe I can keep this to maintain the prior semantics that `@specs_by_name` can include a `nil` spec but perhaps raise only if `e` is type of `DSLError`